uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Reminder.php

```diff
@@ -999,7 +999,7 @@ private static function getUfMatchContact($contact_id) {
    *
    * @return string
    */
-  private static function getCompletedActivitiesExclusionQuery() {
+  private static function _getCompletedActivitiesExclusionQuery() {
```

Comment:
I think its rather better you leave out the underscore, its an old convention used to indicate that a function is private and since the private keyword is used, that is no longer necessary.

Reply:
Done.